### PR TITLE
Docs-ebpf-toc-missing

### PR DESCRIPTION
**Pull request by bmckercher123**
## Description

- "ebfp and EKS" topic missing from TOC
- Merge to nightly and cherrypick to latest

### CODE REVIEW

File: _data/navbars/maintenance.yml
Comment (bcreane) [inline _data/navbars/maintenance.yml:19]: typo - "ePBF" -> "eBPF"

**Review by bcreane — APPROVED** [review]
just fix the typo and squash the commit.

File: _data/navbars/maintenance.yml
Comment (bmckercher123) [reply _data/navbars/maintenance.yml]: !!! Thanks.

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, squash the commits when I merge it, and delete the branch after I've merged it.

**Comment by marvin-tigera** [comment]
I'm sorry but I failed delete the branch after merging the pull request.

**Pull request merged**
